x-pack/plugins/security_solution/public/detections/components/response_actions/get_supported_response_actions.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RESPONSE_SUPPORTED_ACTION_TYPES_IDS } from '../../../../common/constants';
+
+export interface ResponseActionType {
+  id: string;
+  name: string;
+  iconClass: string;
+  description: string;
+}
+
+export const getSupportedResponseActions = (
+  actionTypes: ResponseActionType[],
+  enabledActionsMap: Record<string, boolean>
+): ResponseActionType[] => {
+  return actionTypes.filter((actionType) => {
+    // if (actionType.id === '.osquery' && !enabledActionsMap['.osquery']) {
```

Comment:
Should this just be deleted?

Reply:
I think it may be deleted, it was just turned off recently, but it doesnt make sense to keep it. thanks!